modules/rosa-sts-about-ocm-role.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// rosa_planning/rosa-sts-ocm-role.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-about-ocm-role_{context}"]
+= About the ocm-role IAM role
+
+You need to create the `ocm-role` IAM resource to enable a Red Hat organization of users to create ROSA clusters.
+
+Some considerations for your `ocm-role` IAM resource are:
+
+* You have only one `ocm-role` per Red Hat organization.
+* Any user in a Red Hat organization may create and link an `ocm-role` IAM resource.
+* Only the Red Hat organization administrator can unlink an `ocm-role` IAM resource. This limitation is to protect other Red Hat organization members from disturbing the interface capabilities of other users.
++
+[NOTE]
+====
+If you just created a Red Hat account that is not part of an existing organization, this account is also the organization admin.
+====
++
+* Only one `ocm-role` IAM resource can be created per AWS account per Red Hat organization.
```

Comment:
change to:
> Only one `ocm-role` IAM resource can be _linked_ per AWS account per Red Hat organization. 

There can be more than once ocm-role per AWS account.

Reply:
I see, I think I misunderstood a bit. So, you _can_ have multiple `ocm-role` resources, but you only need [one per the KB article](https://access.redhat.com/articles/6961686)? Is that right @arendej ?

Reply:
Only one ocm-role can be linked to a Red Hat org at a time.
A person can create any number of ocm-roles in AWS.